cost-analyzer/templates/cost-analyzer-ingress-template.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
We also need to check if .Values.ingress here for safety, as some users install with the same values.yaml. See https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/templates/cost-analyzer-network-policy.yaml#L1 for example.

Reply:
This is the standard ingress.yaml file generated by ```helm create```.

I see not other chart that are doing a check on the existence off .Values.ingress. It always exists because it is in the values.yaml

But I can add it if you really want to

Reply:
This is to protect against legacy values.yaml installed via helm --reuse-values.  We'll look to remove support for old values.yaml files in a future release, but for now we're supporting them, so it would be preferable to have this.

Reply:
Done